services/apps/alcs/src/queues/scheduler/scheduler.service.ts

```diff
@@ -9,16 +9,15 @@ export const MONDAY_TO_FRIDAY_AT_2AM_PDT_IN_UTC = '0 0 9 * * 1-5';
 export const EVERYDAY_MIDNIGHT_PST_IN_UTC = '0 0 8 * * *';
```

Comment:
Does this it runs every month on the 8th at midnight?

Reply:
I just noticed that * is Shift + 8. I wonder if this was meant to be `0 0 * * * *`, but the shift was missed by accident?